lib/syslogger.rb

```diff
@@ -18,6 +18,15 @@ class Syslogger
     Logger::UNKNOWN => Syslog::LOG_ALERT
   }
 
+  # Severity label for logging (max 5 chars).
+  SEVERITY_LABEL = {
+    Logger::DEBUG => 'DEBUG',
+    Logger::INFO => 'INFO',
+    Logger::WARN => 'WARN',
+    Logger::ERROR => 'ERROR',
+    Logger::FATAL => 'FATAL',
+  }
+
```

Comment:
:+1:

Connascence of name is a weaker form of connascence than conncascence of position, so this is much better :).